### PR TITLE
added info about local odahuflow 

### PR DESCRIPTION
**Pull request by BPylypenko**
* added prerequisites for local training and packaging in local Quicstart page.
* added a mention about local command in Model Trainings and Model Packagers pages.

### CODE REVIEW

File: source/tutorials_local_wine.rst [hunk 1]
```diff
@@ -10,9 +10,12 @@ a machine learning model locally by leveraging ODAHUFlow's main components.
 Prerequisites
 ~~~~~~~~~~~~~~~~~~~
 
-- :term:`Odahu-flow CLI`
+- Linux based system with bash (at least version 4)
```
Comment (mcdoker18) [inline source/tutorials_local_wine.rst]: It's wrong. Please remove it.

File: source/tutorials_local_wine.rst [hunk 2]
```diff
@@ -10,9 +10,12 @@ a machine learning model locally by leveraging ODAHUFlow's main components.
...
+- Python 3.6 or later
```
Comment (mcdoker18) [inline source/tutorials_local_wine.rst]: This is requirement of `odahuflowctl`, so please remove the line.

File: source/ref_trainings.rst [hunk 3]
```diff
@@ -262,6 +262,22 @@ Getting the model name of the trainings:
...
+
+.. code-block:: bash
+
+    odahuflowctl local train --help
+
+and can run trainings locally:
+
+.. code-block:: bash
+
+    odahuflowctl local train run --help
```
Comment (mcdoker18) [inline source/ref_trainings.rst]: Please, provide a real example of run command

File: source/ref_packagers.rst [hunk 4]
```diff
@@ -121,12 +121,24 @@ Getting the arguments of the packagers:
...
+
+.. code-block:: bash
+
+    odahuflowctl local pack --help
+
+and can run packaging locally:
+
+.. code-block:: bash
+
+    odahuflowctl local pack run --help
```
Comment (mcdoker18) [inline source/ref_packagers.rst]: Please, provide a real example of run command

**Review by mcdoker18 — CHANGES_REQUESTED** [review]

**Review by mcdoker18 — APPROVED** [review]

**Review by maxkochubey — APPROVED** [review]

**Review by vlad-tokarev — APPROVED** [review]

**Pull request merged**